core/src/main/java/org/elasticsearch/indices/IndicesService.java

```diff
@@ -419,9 +406,9 @@ private void removeIndex(String index, String reason, boolean delete) {
             if (delete) {
                 listener.beforeIndexDeleted(indexService);
             }
-            logger.debug("[{}] closing index service (reason [{}])", index, reason);
+            logger.debug("[{}] closing index service (reason [{}])", indexName, reason);
```

Comment:
nit: let's just log the index here (so we get the uuid and such with Nik's latest change)